site/docs/infrastructure-damaged.mdx

```diff
@@ -0,0 +1,56 @@
+
+(optional fields will be omitted from the JSON if there is no value reported)
+
+| field name                             | value                                                                                                  | optional |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------ | -------- |
+| report_date                            | date in YYYY-MM-DD                                                                                     | no       |
+| civic_buildings.destroyed              | cumulative number of civic buildings destroyed to the report date                                      | yes      |
+| ext_civic_buildings.destroyed          | same as `civic_buildings.destroyed` but extrapolated (see below)                                       | no       |
+| educational_buildings.destroyed        | cumulative number of educational buildings (schools, universities, etc) destroyed to the report date   | yes      |
+| ext_educational_buildings.destroyed    | same as `educational_buildings.destroyed` but extrapolated (see below)                                 | no       |
+| educational_buildings.damaged          | cumulative number of educational buildings partially damaged to the report date                        | yes      |                          
+| ext_educational_buildings.damaged      | same as `educational_buildings.damaged` but extrapolated (see below)                                   | no       |
+| mosques.destroyed                      | cumulative number of mosques destroyed to the report date                                              | yes      |
+| ext_mosques.destroyed                  | same as `mosques.destroyed` but extrapolated (see below)                                               | no       |
+| mosques.damaged                        | cumulative number of mosques partially damaged to the report date                                      | yes      |
+| ext_mosques.damaged                    | same as `mosques.damaged` but extrapolated (see below)                                                 | no       |
+| churches.destroyed                     | cumulative number of churches destroyed to the report date                                             | yes      |
+| ext_churches.destroyed                 | same as `churches.destroyed` but extrapolated (see below)                                              | no       |
+| units.destroyed                        | cumulative number of residentional units destroyed to the report date                                  | yes      |
+| ext_units.destroyed                    | same as `units.destroyed` but extrapolated (see below)                                                 | no       |
+
+## Data Sources
+
+For infrastructure damage data, we depend on [Government Media Office (GMO) in Gaza ](https://t.me/s/mediagovps).
```

Comment:
maybe also add that this is reported infrequently:

Reports are provided roughly once every week (as of 2024) and it's unclear how frequently government officials are updating these numbers between reports. For that reason they should be treated as estimates, especially given the magnitude and the difficulty for non-combatants to enter certain regions in Gaza where Israeli forces are present.